packages/elements/src/components/ino-nav-item/readme.md

```diff
@@ -38,11 +38,12 @@ document
 
 ## Properties
 
-| Property       | Attribute       | Description                                                                                                 | Type      | Default     |
-| -------------- | --------------- | ----------------------------------------------------------------------------------------------------------- | --------- | ----------- |
-| `inoActivated` | `ino-activated` | Styles the row in an activated style.  Use this for only one item and to mark it as permantently activated. | `boolean` | `false`     |
-| `inoDisabled`  | `ino-disabled`  | Styles the row in a disabled style.                                                                         | `boolean` | `false`     |
-| `inoText`      | `ino-text`      | The text of this list item.                                                                                 | `string`  | `undefined` |
+| Property           | Attribute            | Description                                                                                                | Type      | Default     |
+| ------------------ | -------------------- | ---------------------------------------------------------------------------------------------------------- | --------- | ----------- |
+| `inoActivated`     | `ino-activated`      | Styles the row in an activated style.  Use this for only one item and to mark it as permanently activated. | `boolean` | `false`     |
+| `inoDisabled`      | `ino-disabled`       | Styles the row in a disabled style.                                                                        | `boolean` | `false`     |
+| `inoSecondaryText` | `ino-secondary-text` | The secondary text of this list item used in a two-lined list.                                             | `string`  | `undefined` |
+| `inoText`          | `ino-text`           | The text of this list item.                                                                                | `string`  | `undefined` |
 
 
```

Comment:
In addition to the Readme it would be great to see its possibilites in inoNav Storybook  examples

Reply:
There actually is an example in the story of the nav-item. Should I add some more?

![image](https://user-images.githubusercontent.com/22963121/105984033-e13bfd80-6099-11eb-9fef-91dd5889807d.png)

Reply:
Did't noticed. That's totally fine